src/main/java/org/example/homework6/Calculator.java

```diff
@@ -0,0 +1,77 @@
+package org.example.homework6;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Calculator {
+    String decisionValue;
+    Scanner scanner = new Scanner(System.in);
+
+    public void calculator() {
+        double firstNum;
+        double secondNum;
+
+        do {
+            System.out.print("Enter a number:");
+            try {
+                firstNum = scanner.nextDouble();
```

Comment:
maybe `Double.parseDouble(scanner.nextLine())` will be more laconic?
according to what you send to me in the chat
in that case you will have to handle NumberFormatException, if you wish